internal/adapter/lsp/document.go

```diff
@@ -194,11 +197,25 @@ func (d *document) DocumentLinkAt(pos protocol.Position) (*documentLink, error)
 	return nil, nil
 }
 
+// Recursive function to check whether a link is within inline code block.
```

Comment:
Tiny detail: as this function checks for _inline code_ exclusively, the comment should reflect that:

` ... a link is within inline code` / `... a link is within inline code / back ticks`

I think the term 'code block' has a general accepted meaning of a (multi-line) code block, and not inline code. It could therefore cause confusion down the track for someone.

Reply:
Yeah, you're totally right. `inline code block` is a bit contradictory...